sjb/config/test_cases/test_branch_openshift_ansible_logging.yml

```diff
@@ -192,11 +31,7 @@ extensions:
         popd
         logging_extras=""
         if [[ "${PULL_REFS%%:*}" == es5.x ]]; then
```

Comment:
you could also add `$EXTRA_EVARS` to your `ansible-playbook` call and pass these in only in the es5.x job, your call

Reply:
We really want to merge es5.x into master (for 3.10) ASAP